src/main/webapp/admin_dashboard.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Admin Dashboard</title>
+		<link rel="stylesheet" href="admin_dashboard_style.css">
+		<script src='https://kit.fontawesome.com/a076d05399.js'></script>
+        <script src="https://www.gstatic.com/charts/loader.js"></script>
+        <script src="admin_dashboard_script.js"></script>
+	</head>    
+	<body>
+		<div class="wrapper">
+			<nav>
+				<header>NeighborGood</header>
+				<ul>
+					<li><a class="active">Dashboard</a></li>
+					<li><a>My Account</a></li>
+					<li><a>Statistics</a></li>
+					<li><a>Settings</a></li>
+					<li><a>Logout</a></li>
+				</ul>
+			</nav>
+
+			<main>
+				<h1>Admin Prototype</h1>
+				<div class="notification-grid">
+					<div>
+						<h3>Notification 1</h3>
+						This thing just happened.
+					</div>
+					<div>
+						<h3>Notification 2</h3>
+						This thing just happened.
+					</div>
+					<div>
+						<h3>Notification 3</h3>
+						This thing just happened.
+					</div>
+				</div>
+
+				<div class="flex-grid">
+                    <div id="card-container">
```

Comment:
Same comment as in the CSS file; could you run this through an HTML formatter? The mixture of tabs and spaces makes the file hard to read if you don't have tabs set to display as four spaces.